app/controllers/dams_resource_controller.rb

```diff
@@ -30,6 +30,17 @@ def show
     # get metadata from solr
     @document = get_single_doc_via_search(1, {:q => "id:#{params[:id]}"} )
 
+    # generate facet collection list for collection page only
+    if !@document.nil?
```

Comment:
Yeah, in this case the 403 seems confusing to me. For example, we may really want status 404 when a records was deleted and we want to do a double check to confirm the deletion.